model/event/host_event.go

```diff
@@ -112,3 +116,22 @@ func LogHostTeardown(hostId, teardownLogs string, success bool, duration time.Du
 func LogMonitorOperation(hostId string, op string) {
 	LogHostEvent(hostId, EventHostMonitorFlag, HostEventData{MonitorOp: op})
 }
+
+// UpdateExecutions updates host events to track multiple executions of the same task
+func UpdateExecutions(hostId, taskId string, execution int) error {
+	const ResourceIdKey = "r_id"
+	const TaskIdKey = "t_id"
+	const DataKey = "data"
+	const ExecutionKey = "execution"
```

Comment:
it's weird to have public values in function scopes. 

also these keys should be declared/checked with bsonutil.MustHaveTag() in the package scope (I hate this, but, alas, it's the pattern. 

also you can cosnt ( <names> = <values>

Reply:
Oops, I was confused about what package I was in when I did this. It should be cleaned up now